backend/danswer/chunking/models.py

```diff
@@ -55,6 +58,34 @@ class IndexChunk(DocAwareChunk):
     embeddings: ChunkEmbedding
 
 
+@dataclass
+class DocMetadataAwareIndexChunk(IndexChunk):
```

Comment:
Note: could also attach this info to the document itself, but this doesn't play as well with our current setup

Reply:
Like in Vespa?